code/framework/extension/src/main/java/io/cattle/platform/extension/impl/ExtensionManagerImpl.java

```diff
@@ -4,23 +4,11 @@
 import io.cattle.platform.extension.ExtensionImplementation;
 import io.cattle.platform.extension.ExtensionManager;
 import io.cattle.platform.extension.ExtensionPoint;
-import io.cattle.platform.util.type.CollectionUtils;
-import io.cattle.platform.util.type.InitializationTask;
-import io.cattle.platform.util.type.NamedUtils;
-import io.cattle.platform.util.type.PriorityUtils;
-import io.cattle.platform.util.type.ScopeUtils;
+import io.cattle.platform.util.type.*;
```

Comment:
Nit: This is the only place in our entire code base where we would use the wildcard on non-static imports. Also, eclipse's import organizer would just re-expand this. Can you not do this.